### ISSUE TITLE
low memory on GPU

### ISSUE THREAD

**Issue by pastaxingxin** [matterport/Mask_RCNN#1702]
Dear all.
I used Mask-RCNN to train my models. For decreasing the training time, I used four GPU to train.
However, the training speed is very slow. 
In addition, I have four GPU, the used memory of GPU is still low. No matter how I changed the  IMAGES_PER_GPU, every GPU only used 100+ MB memory.
![image](https://user-images.githubusercontent.com/17949301/63514371-bb93e780-c4e8-11e9-9fb7-ca77b36e177e.png)

I dont' know how to increase the training speed, and how to increase used memory of each GPU? Could you please help me?
Thanks a lot in advance.

**Comment by xiaoliqiangxxqzz** [comment]
do you solve this problem?

**Comment by IvonaTau** [comment]
Same issue. I guess this is due to memory problem while loading images.

See: https://stackoverflow.com/questions/44563418/low-gpu-usage-by-keras-tensorflow

https://stackoverflow.com/questions/49163592/keras-is-not-using-tensorflow-gpu?rq=1

**Comment by burhr2** [comment]
Hi! kindly see this issue https://github.com/matterport/Mask_RCNN/issues/2312 there is a pointer to the same implementation that supports TensorFlow 2=> (with GPU). If this solves your problem kindly close the issue so as others can navigate to other issues easier
